src/AddTodoForm.js

```diff
@@ -1,17 +1,28 @@
 import React from 'react';
-let AddTodoForm = (props) => {
+
+
+
+let AddTodoForm = ({ onAddTodo }) => {
+    const [todoTitle, setTodoTitle] = React.useState([]);
```

Comment:
Consider making the default value empty string and not empty array for the state variable `todoTitle`